content/community/kug-2023.md

```diff
@@ -109,7 +109,11 @@ A registration fee of $125 will be charged.
 
 ## Accomodation
 
-Albuquerque is NOT an inherently walkable city, however there are two hotel close enough that a car wouldn't be strictly necessary.
+Albuquerque is NOT an inherently walkable city, however there are two decent hotels close enough that a car wouldn't be strictly necessary.
+Besides those there are many good hotels throughout Albuquerque. Any place *east* of the River (i.e. in Albuquerque proper) will be fine and at most 15-20mins by car.
+There is *NO* usable public transport other than ride services.
+
+*We do NOT recommend staying in the nearyby Motel6/Studio6!*
```

Comment:
```suggestion
*We do NOT recommend staying in the nearby Motel6/Studio6!*
```

Reply:
Instead of line 116, how about:

- Please get in touch if there is no availability in the hotels suggested above

Reply:
It is not like we are going to find hotels for them so I'd say no.